src/TagLib/Asf/Tag.cs

```diff
@@ -229,8 +229,8 @@ public string GetDescriptorString (params string [] names)
 		///    The value will be stored in the first value in <paramref
 		///    name="names" /> and the rest will be cleared.
 		/// </remarks>
-		public void SetDescriptorString (string value,
-		                                 params string [] names)
+		public void SetDescriptorString (string value,
+										 params string [] names)
```

Comment:
Sorry, another formatting issue. It should have tabs up to the beginning of the method and then spaces to align the parameter under the first. I think the IDE is trying to be too helpful again :(

If you could fix this that'd be great. It help reduce the size of the code diff and as a result makes review the PR much easier.